tests/org.eclipse.emf.emfstore.test.common/src/org/eclipse/emf/emfstore/client/test/common/util/ServerUtil.java

```diff
@@ -73,6 +74,8 @@
  */
 public final class ServerUtil {
 
+	private static final String EMFSTORE_ACCESSCONTROL_AUTHENTICATION_SUPERUSER_PASSWORD_SALT = "emfstore.accesscontrol.authentication.superuser.password.salt"; //$NON-NLS-1$
+	private static final String EMFSTORE_ACCESSCONTROL_AUTHENTICATION_SUPERUSER_PASSWORD_HASH = "emfstore.accesscontrol.authentication.superuser.password.hash"; //$NON-NLS-1$
```

Comment:
These properties are already available as constants in `ServerConfiguration.java` as `SUPER_USER_PASSWORD_SALT` and `SUPER_USER_PASSWORD_HASH`. Is there a reason they are redefined here?